src/lib/mapping/signMessage.ts

```diff
@@ -1,21 +0,0 @@
-import { signature } from "./signature";
-
-/**
- * Sign the given message. This function assumes (and requires) a user is logged in.
- * @param message The message to sign.
- * @returns The signed version of the message.
- */
-export async function signMessage(
-  data: any,
-  options = { hashAlgorithm: "SHA-256" },
-): Promise<number[]> {
-  const dataToSign = new Uint8Array(data);
```

Comment:
This probably means that passing any string directly to `signMessage` will generate the same `Uint8Array`, so `dataToSign` is always the same no matter the string...